utils/hcl_parse_test.go

```diff
@@ -288,6 +288,44 @@ func TestParseBodyPackageNoCopy(t *testing.T) {
 	}
 }
 
+func TestParseBodyReturnErrorWhenTypeIsUnexpected(t *testing.T) {
+	//Create the HCL byte array
+	hcl := []byte(`base_dir="./"
+	start_port=3000
+	port_increment=1
+	alias=true
+	repository_host="host.com"
+	pims_config_dir="pims_config"
+	pims_dir = "pims"`)
+
+	//Create the parser
+	parser := hclparse.NewParser()
+
+	//Parse the byte array
+	f, diags := parser.ParseHCL(hcl, "config_test")
+
+	//If error it fails
+	if diags.HasErrors() {
+		t.Fatal(diags.Error())
+	}
+
+	//Create a new utility
+	util := NewUtility()
+
+	//Parse the HCL Body
+	_, err := util.ParseBody(f.Body, nil)
```

Comment:
If we pass a `nil` or empty `hcl.Body` object here will the test still pass?

If so, we should do that to make the test shorter and then you can get rid of all the HCL body setup code before line 313.

Reply:
@everettraven 
I see, will test that in my local.

Agree, the test setup is quite long, and this could shorten the test.

Reply:
@everettraven 
It is doable for the empty `hcl.Body`. Already put the required changes.

Thank you.